Libraries/oneMKL/guided_cuRAND_examples_SYCL_migration/sample.json

```diff
@@ -0,0 +1,25 @@
+{
+  "guid": "CB0E93ED-0BB4-44EC-9C0B-924D355B6BC2",
+  "name": "cuRAND Migration",
+  "categories": ["Toolkit/oneAPI Libraries/oneMKL"],
+  "description": "This sample shows the execution of cuRAND function migrated to SYCL.",
+  "toolchain": [ "dpcpp" ],
+  "dependencies": [ "mkl" ],
+  "languages": [ { "cpp": {} } ],
+  "targetDevice": [ "CPU", "GPU" ],
+  "os": [ "linux" ],
+  "builder": [ "cmake" ],
+  "ciTests": {
+  "linux": [{
+    "steps": [
+      "mkdir build",
+      "cd build",
+      "cmake ..",
+      "make",
+      "make run_amax"
```

Comment:
@IgorOchocki Should this be ``make run_mt19937_uniform`` ? ``make run_amax` doesn't seem to exist